datasets/sentinel-products-ca-mirror.yaml

```diff
@@ -0,0 +1,30 @@
+Name: Sentinel Products Canada Mirror
+Description: "The official Government of Canada 🍁 mirror for Sentinel-1,2,3 data under the EU's Copernicus programme focused on Canada-wide coverage. Additional Sentinel-2 and Sentinel-6 products over Canada are retrieved from [EUMETSAT](https://eoportal.eumetsat.int/cas/login)"
+Documentation: https://roda.sentinel-hub.com/sentinel-s1-l1c/GRD/readme.html
+Contact: eodms-sgdot@nrcan-rncan.gc.ca
+ManagedBy: "[Natural Resources Canada](https://www.nrcan.gc.ca/)"
+UpdateFrequency: Sentinel-1 is a near real-time (NRT) dataset retrieved from ESA's [collaborative hubs](https://colhub.copernicus.eu/), not the open hub. Other sensors are updated as quickly as possible based on Canada coverage availability at the source.
+Collabs:
+  ASDI:
+    Tags:
+      - satellite imagery
+Tags:
+  - agriculture
+  - earth observation
+  - satellite imagery
+  - geospatial
+  - sustainability
+  - disaster response
+  - synthetic aperture radar
+License: Access to Sentinel data is free, full and open for the broad Regional, National, European and International user community. View [Terms and Conditions](https://scihub.copernicus.eu/twiki/do/view/SciHubWebPortal/TermsConditions).
+Resources:
+  - Description: Sentinel data in an S3 bucket broken down by sensor, product type and date.
+    ARN: arn:aws:s3:::sentinel-products-ca-mirror
+    Region: ca-central-1
+    Type: S3 Bucket
+DataAtWork:
+  Tools & Applications:
+    - Title: A STAC service for Sentinel Products on the Canada Mirror
+      URL: https://eodms-sgdot.nrcan-rncan.gc.ca/stac/collections
```

Comment:
link isn't public and also it will be redundant information if you add the STAC URL in the Resource section.